core/test/app_grc.cpp

```diff
@@ -13,13 +13,45 @@
 
 namespace grg = gr;
 
+template<typename T>
+struct ArraySource : public grg::Block<ArraySource<T>> {
```

Comment:
boy-scout-rule: could you also rename `grg` -> `gr`

Reply:
Done